doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
I'm so thoroughly confused since save is usually synonymous with retain and yet it's apparently not in this case and I'm not quite sure what `it is a matter of how many fewer digits they can have if the offset is used.` means in a practical sense.